### DISCUSSION TITLE
Is there a command to see reasoning for strikes?

### DISCUSSION THREAD

**Discussion by oscie57** [jagrosh/Vortex#89] [Questions] [answered]
Well, is there? I have a user who has multiple strikes and they want me to pardon them, but the log channel was deleted at one point so I cant go and check there to see the reasons. Is there a command for it? (maybe if not it could be a possible suggestion). Thanks to who ever is reading (and possibly replying to) this!

**Comment by jagrosh** [comment, marked as answer]
The reasons are only stored in the log channel, so if that channel is deleted, the reasons are gone. You can see all the data Vortex keeps about your server via the `>>export` command.
